revolv/payments/tests/services/test_payment_service.py

```diff
@@ -3,7 +3,7 @@
 from django.test import TestCase
 from revolv.payments.models import PaymentInstrumentType
 from revolv.payments.services import PaymentService, PaymentServiceException
-from revolv.project.tests import CreateTestProjectMixin
+from revolv.project.tests.tests import CreateTestProjectMixin
```

Comment:
lol can we rename the innermost tests file to something like test_general or something? project.tests.tests is a bit redundant